openmc/mgxs_library.py

```diff
@@ -908,10 +935,12 @@ def set_multiplicity_mgxs(self, nuscatter, scatter, nuclide='total',
                     ['universe', 'cell', 'material'])
 
         if self._representation is 'isotropic':
```

Comment:
Is there an `XSdata.representation` property getter? If so, use it here and elsewhere.